reverse_engineering/api.js

```diff
@@ -79,6 +86,10 @@ module.exports = {
 						isEmpty: dbCollections.length === 0,
 					});
 				} catch (error) {
+					if (connectionInfo.databaseName) {
```

Comment:
if we select database explicitly, then we log the error but mute it and it gives the impression the connection is successful, in this case I just pass error throug